Content/Tiles/BossDomain/RoyalHoneyClumpTile.cs

```diff
@@ -0,0 +1,68 @@
+﻿using PathOfTerraria.Common.Systems.DisableBuilding;
+using PathOfTerraria.Content.Items.BossDomain;
+using System.Collections.Generic;
+using Terraria.GameContent;
+using Terraria.ID;
+
+namespace PathOfTerraria.Content.Tiles.BossDomain;
+
+internal class RoyalHoneyClumpTile : ModTile, ICanCutTile
+{
+	public override void SetStaticDefaults()
+	{
+		Main.tileCut[Type] = true;
+		Main.tileSolid[Type] = false;
+		Main.tileFrameImportant[Type] = true;
+
+		DustType = DustID.Honey;
+
+		AddMapEntry(new Color(255, 156, 12));
+	}
+
+	public override void HitWire(int i, int j)
+	{
+		Wiring.SkipWire(i, j);
+		Tile tile = Main.tile[i, j];
+
+		if (tile.TileFrameY == 0)
+		{
+			tile.TileFrameY = 18;
+
+			if (Main.netMode == NetmodeID.Server)
+			{
+				NetMessage.SendTileSquare(-1, i, j);
+			}
+		}
+	}
+
+	bool ICanCutTile.CanCut(int i, int j)
```

Comment:
Should this have a access level?

Reply:
For component hooks like this I usually make it explicitly part of the interface; it doesn't need an access modifier and it makes it clear that it's part of ICanCutTile and not a ModTile hook.